cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,30 @@
+class PaymentStepPage {
+    private paymentT: string;
+    private paymentC: string;
+    private confirmationO: string;
+    // private confirmationOrder: string;
+
+
+    constructor() {
+        this.paymentT = "a.bankwire";
+        this.paymentC = ".cart_navigation span";
+        this.confirmationO = "#center_column > div > p > strong";
+    }
+
+    public PaymentType(): void {
+        cy.get(this.paymentT).click()
+    }
+
+    public ConfirmPayment(): void {
+        cy.get(this.paymentC).click()
+    }
+
+
+    public ConfirmationMessage(): Chainable<JQuery<HTMLElement>> {
```

Comment:
Here you can call it GetConfirmationMessage, applying the rule to use verbs in infinitive mode

Reply:
change to GetConfirmationMessage